Dockerfile

```diff
@@ -55,10 +57,13 @@ RUN chown 1000:1000 /jupyter
 COPY --chown=1000:1000 . /jupyter
 
 # Switch to non-root user. It is crucial for security reasons to not run jupyter as root user!
+RUN chmod -R 777 /tmp
```

Comment:
Are we sure we need this? /tmp should have proper permissions already and mucking with /tmp permissions is a known way to introduce security holes.

Also, if we do need this, let's not put the command right between the comment and the command the comment is talking about.

Reply:
I left this in after testing :sweat_smile:. deleting it since it's no longer used